pkg/antctl/raw/traceflow/command.go

```diff
@@ -257,6 +257,12 @@ func parseFlow() (*v1alpha1.Packet, error) {
 		}
 		pkt.TransportHeader.TCP.DstPort = int32(r)
 	}
+	if r, ok := fields["tcp_flags"]; ok {
+		if pkt.TransportHeader.TCP == nil {
+			pkt.TransportHeader.TCP = new(v1alpha1.TCPHeader)
+		}
+		pkt.TransportHeader.TCP.Flags = int32(r)
```

Comment:
I don't think so. Agent controller just injects everything in the CRD into the Traceflow packet.

Reply:
Sorry but when we generate the TF packet, if it is not TCP packet, how we handle fields TCP flags?

Understood maybe this is not relevant to your PR.

Reply:
I think it will not be set. When `tf.Spec.Packet.IPHeader.Protocol` is set to 6 (TCP), TCP flags will be set in packet-out.
https://github.com/vmware-tanzu/antrea/blob/master/pkg/agent/openflow/client.go#L695

Reply:
Ok. Might be better to log a warning, or even return an error. But not a big deal I think.